src/ensembl/src/content/app/custom-download/containers/content/CustomDownloadContent.tsx

```diff
@@ -88,7 +84,6 @@ const mapDispatchToProps: DispatchProps = {
 
 const mapStateToProps = (state: RootState): StateProps => ({
   showSummary: getShowPreviewResult(state),
-  launchBarExpanded: getLaunchbarExpanded(state),
   showExampleData: getShowExampleData(state)
 });
 
```

Comment:
Can we switch to using hooks for this component?

Reply:
In due course, when we resume work on custom download. I imagine it won't be for a while.